psychopy/experiment/components/mouse/__init__.py

```diff
@@ -149,7 +149,7 @@ def _writeClickableObjectsCodeJS(self, buff):
         code = (
             "// check if the mouse was inside our 'clickable' objects\n"
             "my.gotValidClick = false;\n"
-            "for (const obj of [{clickable}]) {{\n"
+            "for (const obj of {clickable}) {{\n"
             "  if (obj.contains({name})) {{\n"
             "    my.gotValidClick = true;\n"
             "    {name}.clicked_name.push(obj.name);\n"
```

Comment:
This is the line that I was saying before should say something like `clicked_{param}` 
Is there a reason that you've diverged from the Python equivalent function?

Reply:
No, but the fix is on its way.